geode-core/src/main/java/org/apache/geode/internal/statistics/StatMonitorHandler.java

```diff
@@ -228,7 +222,7 @@ private void work() {
             }
           }
           if (working && latestTask != null) {
-            List<StatisticsMonitor> currentMonitors = StatMonitorHandler.this.monitors;
+            ConcurrentHashSet<StatisticsMonitor> currentMonitors = StatMonitorHandler.this.monitors;
```

Comment:
I don't think this copy is needed since `monitors` is never updated after initialization.